### DISCUSSION TITLE
How to visualize \uxxxx

### DISCUSSION THREAD

**Discussion by valsamis-d** [JanDeDobbeleer/oh-my-posh#4121] [General]
Hello,

I'm a newbie to oh-my-posh and I love it!

I'm looking at the themes by loading them in Visual Studio Code and would like to know if there's a way to visualize a glyph given its \uxxx notation. For example:

```
            "charged_icon": "\ue22f ",
            "charging_icon": "\ue234 ",
            "discharging_icon": "\ue231 "
```
Thank you
Dimitrios

**Comment by JanDeDobbeleer** [comment]
@valsamis-d when using PowerShell you should be able to use `Write-Host "\uxxxx"`, every shell is a bit different in that regard.

**Comment by valsamis-d** [comment]
Thanks for the response @JanDeDobbeleer ! Tried that but in the output I get "\uxxxx"

**Reply by JanDeDobbeleer** [reply]
You can actually easily use the Need Font [cheat sheet](https://www.nerdfonts.com/cheat-sheet) with the xxxx part (e.g. `e22f`).

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/2492783/2fdd5f44-6d16-40a4-b6b6-aa9a2cbccfda)

**Reply by valsamis-d** [reply]
Awesome! Thanks!
